scripts/pylib/twister/twisterlib.py

```diff
@@ -730,10 +730,14 @@ def handle(self):
         ser_pty_process = None
         if serial_pty:
             master, slave = pty.openpty()
+            serial_cmd = re.split(',| ', serial_pty)
+            # We try to pass the build directory to the device serial pty for specially purpose
```

Comment:
> for specially purpose
you need to be more specific here.

Reply:
Thanks for the review!  @nashif , do you think the updated description is appropriate or not?